src/resources/resources.json

```diff
@@ -603,6 +603,32 @@
       "redirectMessage": "Sie werden in {seconds, plural, =1 {# Sekunde} other {# Sekunden}} auf die <0>Anmeldeseite</0> weitergeleitet."
     }
   },
+  {
+    "namespace": "roomMembershipConfirmation",
+    "language": "en",
+    "resources": {
+      "confirmationSuccess": "Congratulations! You are now a member of room <0>{roomName}</0>.",
+      "invitationConfirmation": "Please click on the button in order to join room <0>{roomName}</0>.",
+      "invitationConfirmationDisclaimer": "By clicking on the button you give your consent to ???",
```

Comment:
We gotta make a ticket to fill in this disclaimer I guess 🤔

Reply:
Yes, we have to get inspiration from project management

Reply:
Created le ticket (https://educandu.atlassian.net/browse/EDU-214)